app/models/sample.rb

```diff
@@ -226,7 +226,7 @@ def initiate_s3_cp
     return unless status == STATUS_CREATED
     stderr_array = []
     total_reads_json_path = nil
-    max_lines = PipelineRun::MAX_INPUT_FRAGMENTS * 4
+    max_lines = 4 * (max_input_fragments ? max_input_fragments : PipelineRun::DEFAULT_MAX_INPUT_FRAGMENTS)
```

Comment:
“max_lines = 4 * (max_input_fragments || PipelineRun::DEFAULT_MAX_INPUT_FRAGMENTS)” ? 
Similar simplification for other places (both in .rb and .jsx files)